Automattic/wp-calypso
# Add Beta Testing page

#### Changes proposed in this Pull Request

More info: pd2n8V-46-p2

This adds the Beta Testing page which showcases info about the Horizon programme.
The menu link is added to the Public API via D63456-code and #54588 (in the fallback menu).

Proposed design:
<img width="480" alt="beta-testing-screen-v8" src="https://user-images.githubusercontent.com/2749938/126165669-11b2e27c-c2ab-4487-ab66-64b7e57d0481.jpg">

#### Testing instructions

* Go to the /beta-testing page
* The page should render properly on any screen size

<img width="480" alt="Screenshot on 2021-07-21 at 14-05-25" src="https://user-images.githubusercontent.com/2749938/126480958-98b70ef4-f967-4f8f-b95e-65c20759fc93.png">
<img width="320" alt="Screenshot on 2021-07-21 at 14-05-48" src="https://user-images.githubusercontent.com/2749938/126480954-672b45f9-28bc-4937-b85c-a97b08001f0d.png">
<img width="249" alt="Screenshot on 2021-07-21 at 14-08-28" src="https://user-images.githubusercontent.com/2749938/126480951-b269a30d-bbb8-442e-af86-e9d72f7f0933.png">

* The `Join the Beta Program` link should go to horizon.wordpress.com
* The `Learn how it works` link should go to the About page on horizonfeedback.wordpress.com
* The `Recent Beta Features` should populate with the last 3 posts from horizonfeedback.wordpress.com

* There should be no related JS errors
